composer.json

```diff
@@ -9,7 +9,8 @@
 
         "league/tactician": "^1.0",
         "ramsey/uuid": "^3.0",
-        "coduo/php-to-string": "^2.0"
+        "coduo/php-to-string": "^2.0",
+        "doctrine/dbal": "^2.5"
```

Comment:
For now I would leave it here. 
But as soon as we have more than this one bridge, we should extract this into separate package.